### DISCUSSION TITLE
If it is possible to merge the table of content (for one page) together with the primary content

### DISCUSSION THREAD

**Discussion by xihajun** [squidfunk/mkdocs-material#4821] [Q&A] [answered]
It seems that `md-sidebar--secondary` and `md-sidebar--primary` are controlling the primary content (global), and table of content (page). So we will have two content: one on the left, the other on the right.

I wonder if it is possible to merge them together on the left, like the thing shows in readthedocs.

<img width="282" alt="image" src="https://user-images.githubusercontent.com/25631641/210358931-03f2cb59-8e92-4ccb-9b13-555edeb823c8.png">

**Comment by squidfunk** [comment, marked as answer]
Yes, the feature flag is called [`toc.integrate`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-integration):

``` yaml
theme:
  features:
    - toc.integrate 

```

**Reply by xihajun** [reply]
thank you so much

**Reply by andrzejdoro** [reply]
Is it possible to hide/expand integrated table of contents?

**Reply by squidfunk** [reply]
Yes, see https://github.com/squidfunk/mkdocs-material/issues/2162#issuecomment-752917362

**Reply by andrzejdoro** [reply]
Sorry, I used the wrong word. What I meant was, is it possible to collapse/expand the integrated table of contents (by clicking something like "+")? Read the Docs has this feature (https://docs.readthedocs.io/en/stable/index.html).

**Reply by squidfunk** [reply]
No, that's not possible.
